tree/ntuple/v7/src/RNTupleProcessor.cxx

```diff
@@ -192,6 +191,7 @@ ROOT::Experimental::RNTupleJoinProcessor::RNTupleJoinProcessor(const RNTupleOpen
                                                                std::unique_ptr<RNTupleModel> model)
    : RNTupleProcessor({mainNTuple}), fJoinFieldNames(joinFields)
 {
+   fNTuples.emplace_back(mainNTuple);
```

Comment:
Hm, isn't the `mainNTuple` already placed in `fNTuples` the constructor of `RNTupleProcessor`? (edit: ah, it's removed in the next commit, never mind)